api-tests/specs/assign-roles/assign-roles.js

```diff
@@ -1,9 +1,9 @@
 const requests = require('../../data/assign-roles/assign-roles')
 const conf = require('../../config/conf')
-const sitesEndpointUri = ':8083/sites';
-const rolesEndpointUri = ':8082/roles';
-const practitionerEndpointUri = ':8081/practitioner'
-let assignRoleEndpointUri = ':8081/practitioner/{personId}/roles'
+const sitesEndpointUri = ':8072/api/sites';
+const rolesEndpointUri = ':8072/api/roles';
+const practitionerEndpointUri = ':8072/api/practitioner'
+let assignRoleEndpointUri = ':8072/api/practitioner/{personId}/roles'
```

Comment:
won't it be easier/simpler for the gateway to be on 8080?

Reply:
changed